django_airavata/apps/workspace/static/django_airavata_workspace/js/components/experiment/ExperimentEditor.vue

```diff
@@ -237,29 +242,41 @@ export default {
     uploadInputFiles: function() {
       let uploads = [];
       this.localExperiment.experimentInputs.forEach(input => {
-        if (
-          input.type === models.DataType.URI &&
-          input.value &&
-          input.value instanceof File
-        ) {
-          let data = new FormData();
-          data.append("file", input.value);
-          data.append("project-id", this.localExperiment.projectId);
-          data.append("experiment-name", this.localExperiment.experimentName);
-          let uploadRequest = apiUtils.FetchUtils.post(
-            "/api/upload",
-            data
-          ).then(result => (input.value = result["data-product-uri"]));
-          uploads.push(uploadRequest);
-        }
-      });
-      return Promise.all(uploads);
+              if (
+                input.type === models.DataType.URI &&
+                input.value &&
+                input.value instanceof File
+              ) {
+                let data = new FormData();
+                data.append("file", input.value);
+                data.append("project-id", this.localExperiment.projectId);
+                data.append("experiment-name", this.localExperiment.experimentName);
+                let uploadRequest = apiUtils.FetchUtils.post(
+                  "/api/upload",
+                  data
+                ).then(result => (input.value = result["data-product-uri"]));
+                uploads.push(uploadRequest);
+              }
+            });
+            return Promise.all(uploads);
+      // if(this.userfiles[this.input_name]){
```

Comment:
Remove these commented out lines. Also it would be nice to undo the indentation change above.